tests/autorag/evaluate/metric/test_retrieval_metric.py

```diff
@@ -19,26 +19,26 @@
     ['test-9', 'pred-0', 'pred-8', 'pred-9'],  # recall: 1.0, precision: 0.25, f1: 2/5
     ['test-13', 'test-12', 'pred-10', 'pred-11'],  # recall: 2/3, precision: 0.5, f1: 4/7
     ['test-14', 'pred-12'],  # recall: 1.0, precision: 0.5, f1: 2/3
-    ['pred-13'],  # recall: 0.0, precision: 0.0, f1: 0.0
+    ['pred-13'],  # retrieval_gt is empty so not counted
 ]
 
 
 def test_retrieval_f1():
-    solution = [0.5, 2 / 7, 2 / 5, 4 / 7, 2 / 3, 0.0]
+    solution = [0.5, 2 / 7, 2 / 5, 4 / 7, 2 / 3]
```

Comment:
Why the solution's last elements is not `None`
I think the result will return `None` if the retrieval_gt is empty.